package.json

```diff
@@ -16,8 +16,11 @@
     "build": "ember build",
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
+    "lint:js:fix": "eslint . --fix",
```

Comment:
Also needs a `lint:fix` script, and a tweak to `lint`. 

From https://github.com/emberjs/rfcs/blob/pretty-please!/text/0628-prettier.md#packagejson-scripts:

```json
{
  "scripts": {
    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel 'lint:!(fix)'",
    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
    "lint:js:fix": "eslint . --fix",
    "lint:hbs:fix": "ember-template-lint . --fix"
  }
}
```

Reply:
Perfect; I’ll add that. Thank you!